src/jit/importer.cpp

```diff
@@ -1131,18 +1133,28 @@ GenTreePtr Compiler::impAssignStructPtr(GenTreePtr      dest,
         }
         else
         {
+            // Case of call returning a struct in register(s)
+
```

Comment:
I would reword this as "Case of a call returning a struct in one or more registers." (Also, include a period as per CC).